httpcore5/src/main/java/org/apache/hc/core5/reactor/ssl/SSLIOSession.java

```diff
@@ -345,6 +356,7 @@ private void updateEventMask() {
         }
         if (this.status == CLOSING && !this.outEncrypted.hasData()) {
             this.sslEngine.closeOutbound();
+            this.outboundClosedCount.incrementAndGet();
```

Comment:
How likely is it to overflow here?

Reply:
@michael-o I would say unlikely. Fundamentally we only care if it is greater than zero, so it could well be a bool.

Reply:
Then make it a boolen?

Reply:
@michael-o What for?

Reply:
If you say it could be a boolean why not make into a boolean instead of an int.

Reply:
@michael-o It could be but the number of invocations of  `#closeOutpund` might also prove useful when debugging. What would be a benefit of using bool here?